packages/asc-ui/src/components/Blog/BlogBody/BlogBody.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react'
 import BlogBodyStyle, { Props } from './BlogBodyStyle'
 
-const BlogBody: React.FC<Props> = ({ children }) => (
-  <BlogBodyStyle>{children}</BlogBodyStyle>
-)
+const BlogBody: React.FC<Props & React.HTMLAttributes<HTMLElement>> = ({
```

Comment:
Replace HTMLAttributes type, which is huge with a small custom type/interface containing just the className